phase2/addon/lib/timer.js

```diff
@@ -426,15 +419,15 @@ const debug = {
 
                 timerData.elapsedTime = Number(subArgs[2]);
                 return "elapsedTime set to " + Number(subArgs[2]);
-              break;
+              // break;
               case "tick_length_s":
                 if (!subArgs[2])
                   return "error: invalid use of time set tick_length_s command.";
 
                 prefs["timer.tick_length_s"] = Number(subArgs[2]);
                 return "new tick length in seconds: " + prefs["timer.tick_length_s"];
-                break;
-              case "st-diff":
+                // break;
+              case "st-diff": {
```

Comment:
putting the case code in a block solves the issue with declaring locally scoped variables but makes the code look inconsistent.